cex-collection/js-example/index.js

```diff
@@ -3,8 +3,11 @@ import {ethers} from 'ethers'
 import {PaymasterClient, SponsorClient, WhitelistType} from 'megafuel-js-sdk'
 
 async function cexDoGaslessTransferTx() {
+  const chainID = process.env.CHAIN_ID
+  const privatePolicyUUID = process.env.PRIVATE_POLICY_UUID
+  const sponsorUrl = process.env.SPONSOR_URL
   // Provider for sending the transaction (e.g., could be a different network or provider)
-  const paymasterClient = new PaymasterClient(process.env.PAYMASTER_URL)
+  const paymasterClient = PaymasterClient.newPrivatePaymaster(sponsorUrl+"/"+chainID, privatePolicyUUID, undefined, {staticNetwork: ethers.Network.from(Number(chainID))})
```

Comment:
what is the `undefined` here, this is really not a good style.. can we improve this?

Reply:
updated to `const paymasterClient = PaymasterClient.newPrivatePaymaster(sponsorUrl, policyUUID,  ethers.Network.from(Number(chainID)), {staticNetwork: ethers.Network.from(Number(chainID))})
  `